package.json

```diff
@@ -17,12 +17,13 @@
     "build:packages": "node ./bin/packages/build.js",
     "build:core": "cross-env NODE_ENV=production webpack",
     "build": "npm run build:packages && npm run build:core",
-    "build:release": "./bin/build-plugin-zip.sh",
+    "build:release": "cross-env WC_ADMIN_PHASE=plugin ./bin/build-plugin-zip.sh",
+    "build:php": "php bin/generate-feature-config.php",
```

Comment:
I originally went with `build:php` thinking we might add something else in the future, but I agree with you -- since this doesn't really build PHP files/parts of the app and just config, I think `build:feature-config` sounds good.

Reply:
Updated in https://github.com/woocommerce/wc-admin/pull/1500/commits/45badff93d44a9920192796be5875bab131c0a93.